### PR TITLE
Feature/zainnk902/connect class api to frontend

### PR DESCRIPTION
**Pull request by zainnk902**
### Administrative Info

Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Github Issue]-[3-4 word description separated by dashes]`.

### What issue(s) does this branch close?
#50 - Connecting Class API to Frontend
### Changes

What changes did you make?

- I added a "getAllClasses" api route to get all the classes from the database, which is called on the Classes tab on the League page.
- Also made script to

### Testing

How did you confirm your changes worked?

- I made a script that adds classes to the database, and the classes added always rendered on the page
- When I didn't add any classes or deleted all classes from the database, the page would show "There are no Classes".

### Confirmation of Change

Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change:

Classes rendered when script populates database with classes:
Screen Shot 2022-04-04 at 1 36 58 AM

"There are no Classes" rendered when script deletes all classes from the database:
Screen Shot 2022-04-04 at 1 38 22 AM

### CODE REVIEW

File: lib/database/classes.ts [hunk 1]
```diff
@@ -77,5 +81,24 @@ const getClass = async (id: string): Promise<Class | null> => {
 
   return classes;
 };
+const getAllClasses = async (): Promise<Class[]> => {
+  const query = {
+    text: "SELECT e.name, c.event_information_id, c.min_level, c.max_level, c.rrstring, c.start_time, c.end_time, c.language FROM event_information e, classes c WHERE e.id = c.event_information_id AND e.type = 'class'",
```
Comment (Anshul-Birla) [inline lib/database/classes.ts:86]: In the database, these are being saved as a "Class" not "class". So select where type='Class'

File: lib/database/classes.ts [hunk 2]
```diff
@@ -77,5 +81,24 @@ const getClass = async (id: string): Promise<Class | null> => {
...
+  };
+
+  const res = await client.query(query);
+  // if (res.rows.length == 0) {
+  //   return null;
+  // }
```
Comment (Anshul-Birla) [inline lib/database/classes.ts:92]: delete

**Review by Anshul-Birla — APPROVED** [review]
Perfect! LGTM

**Review by ryanbuui — APPROVED** [review]
LGTM! Nice job.

**Pull request merged**